server/plugin.go

```diff
@@ -8,6 +8,7 @@ import (
 	"github.com/mattermost/mattermost-server/plugin"
 )
 
+// Plugin is the object to run the plugin
```

Comment:
nit: I'm not fond of the phrase `object to run the plugin`. Perhaps something like:
```suggestion
// Plugin implements the interface expected by the Mattermost server to communicate between the server and plugin processes.
```